modules/admin/admin-ui/src/main/resources/web/spec/common/form/set/itemset/_spec.ts

```diff
@@ -0,0 +1,2 @@
+///<reference path='FormItemSet.spec.ts' />
+///<reference path='FormItemSetView.spec.ts' />
```

Comment:
This will not get specs to the common specs file. What's missing is a new line in /form/set/_spec.ts:
///<reference path='optionset/_spec.ts' />
///<reference path='itemset/_spec.ts' />